presto-main/src/main/java/io/prestosql/security/GroupProviderManager.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.security;
+
+import java.util.Set;
+
+public interface GroupProviderManager
```

Comment:
We can remove this interface for now, since it's identical to the one in the SPI. If we need something different later, it's easy to refactor without breaking anyone.

After removing this interface, we can rename `PrestoGroupProviderManager` to just `GroupProviderManager` which is more in line with others, like `PasswordAuthenticatorManager`. It would implement `GroupProvider` from the SPI.